sass/color/_config.scss

```
@@ -24,6 +26,17 @@ $contrast-light: white !default;
 /// @group color-config
 $contrast-dark: black !default;
 
+// Contrast Settings
+// --------------
+///
```

```suggestion
/// Access the configurable contrast variables as an Accoutrement token map.
/// @access private
```

I don't think we need to expose this as part of the public API, it's really meant for internal use.

---

made this private